models/block.go

```diff
@@ -20,7 +20,10 @@ var (
 	EarliestBlockNumber  = big.NewInt(0)
 )
 
-var GenesisBlock = &Block{Block: types.GenesisBlock}
+var GenesisBlock = &Block{
+	Block:             types.GenesisBlock,
+	TransactionHashes: []gethCommon.Hash{},
```

Comment:
what does this change affect then with block transactions?

Reply:
I think this is how default values for interfaces work, in Golang. If it doesn't have a concrete value, it will assign `null`.

Reply:
I understand that, but why is it an issue? because of some serialization that it changes or?

Reply:
Yeah, this issue here https://github.com/onflow/flow-evm-gateway/issues/371#issuecomment-2262828643. serialization related.